frontend/src/metabase/visualizations/visualizations/SmartScalar/utils.ts

```diff
@@ -73,42 +63,3 @@ export const getValueHeight = (height: number): number => {
 export const getChangeWidth = (width: number): number => {
   return Math.max(width - ICON_SIZE - ICON_MARGIN_RIGHT - 2 * SPACING, 0);
 };
-
-export const getFittedPreviousValue = ({
-  width,
-  change,
-  previousValueCandidates,
-  fontFamily,
-}: {
-  width: number;
-  change: string;
-  previousValueCandidates: (string | string[])[];
-  fontFamily: string;
-}): {
-  isPreviousValueTruncated: boolean;
-  fittedPreviousValue?: string | string[];
-} => {
-  const changeWidth = measureText(change, {
-    size: "1rem",
-    family: fontFamily,
-    weight: 900,
-  }).width;
-
-  const availablePreviousValueWidth =
-    getWidthWithoutSpacing(width) -
-    (changeWidth + 2 * SPACING + ICON_SIZE + ICON_MARGIN_RIGHT);
-
-  const matchIdx = previousValueCandidates.findIndex(previousValue => {
-    const previousValueWidth = measureText(innerText(previousValue), {
-      size: "0.875rem",
-      family: fontFamily,
-      weight: 700,
-    }).width;
-    return availablePreviousValueWidth >= previousValueWidth;
-  });
-
-  return {
-    isPreviousValueTruncated: matchIdx !== 0,
-    fittedPreviousValue: previousValueCandidates[matchIdx],
-  };
-};
```

Comment:
Everything this function was doing is now inlined in a shorter way here:
https://github.com/metabase/metabase/pull/35184/files#diff-b0fbfd8b0eb7fcb57feb5b0b2df90ad1925947bac9fc8aefab49fbd15dfbc76aR106-R128